### PR TITLE
Update doc for mvp2

### PR DESCRIPTION
**Pull request by Nicogene**
This PR update the documentation in light of mvp2 and remove twolinks asm that now is not needed anymore.

After this we can tag the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Generate URDF model from CREO mechanisms.
...
 
  <img src="https://github.com/icub-tech-iit/creo2urdf/assets/19152494/19866e6c-8f52-4010-84ef-db458189b753" width=600 align=center >
  <img src="https://github.com/icub-tech-iit/creo2urdf/assets/19152494/968a544f-d991-423f-9b4f-752d0db9dd45" width=600 align=center >
 
 ## Assumptions & limitations
 
-- The assembly must contains max 2 links an 1 revolute joint.
-- The joint is in 0.
+- The assembly must only one not circular kinematic chain.
+- The joint are in 0.
```
Comment (pattacini) [inline README.md:19]:
```suggestion
- The joints are in 0.
```

File: README.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ Generate URDF model from CREO mechanisms.
...
+**Right now this plugin support only for creating the urdf of the ergocub head**
 
  <img src="https://github.com/icub-tech-iit/creo2urdf/assets/19152494/19866e6c-8f52-4010-84ef-db458189b753" width=600 align=center >
  <img src="https://github.com/icub-tech-iit/creo2urdf/assets/19152494/968a544f-d991-423f-9b4f-752d0db9dd45" width=600 align=center >
 
 ## Assumptions & limitations
 
-- The assembly must contains max 2 links an 1 revolute joint.
-- The joint is in 0.
+- The assembly must only one not circular kinematic chain.
```
Comment (pattacini) [inline README.md:18]: What do you mean?

Comment (pattacini) [reply README.md:18]:
```suggestion
- The assembly must contain only one non-circular kinematic chain.
```

**Review by pattacini — APPROVED** [review]

**Comment by Nicogene** [comment]
[`d7a060d` (#34)](https://github.com/icub-tech-iit/creo2urdf/pull/34/commits/d7a060d209fa56f0e76e5e105671d103be9146a3) removes all the twolinks references, since now w/ mvp2 it is not needed anymore

cc @pattacini @mfussi66

**Pull request merged**